<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Unexpected I/O log</title>
<meta http-equiv="cache-control" content="no-cache"></meta>
<meta http-equiv="content-type" content="text/html; charset=utf-8"></meta>
<link rel="stylesheet" href="../../ct_default.css" type="text/css"></link>
<script type="text/javascript" src="../../jquery-latest.js"></script>
<script type="text/javascript" src="../../jquery.tablesorter.min.js"></script>
</head>
<body>

<br />
<h3>Unexpected I/O</h3>
<pre>

</pre>
<div class="error_logger"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-04-29 21:56:22.954 ***</b>
=ERROR REPORT==== 29-Apr-2024::21:56:22.954071 ===
** Generic server my_test_name terminating 
** Last message in was stop
** When Server state == {formatted,[]}
** Reason for termination ==
** stopped
** Client <0.244211.0> stacktrace
** [{gen_server,mc_recv,5,[{file,"gen_server.erl"},{line,1805}]},
    {gen_server,multi_call,4,[{file,"gen_server.erl"},{line,1766}]},
    {gen_server_SUITE,multicall,1,[{file,"gen_server_SUITE.erl"},{line,1487}]},
    {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1794}]},
    {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1303}]},
    {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,1235}]}]


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-04-29 21:56:22.954 ***</b>
=CRASH REPORT==== 29-Apr-2024::21:56:22.954676 ===
  crasher:
    initial call: gen_server_SUITE:init/1
    pid: <0.244215.0>
    registered_name: my_test_name
    exception exit: stopped
      in function  gen_server:handle_msg/6 (gen_server.erl, line 2254)
    ancestors: [<0.244211.0>]
    message_queue_len: 0
    messages: []
    links: [<0.244211.0>]
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 6882
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-04-29 21:57:02.542 ***</b>
=ERROR REPORT==== 29-Apr-2024::21:57:02.542541 ===
** Generic server hurra terminating 
** Last message in was {'EXIT',&lt;0.248562.0&gt;,
                               {#Ref&lt;0.1054348712.1589641217.164424&gt;,1008030,
                                ok,
                                [{gen_server_SUITE,spec_init}],
                                []}}
** When Server state == {formatted,{}}
** Reason for termination ==
** {#Ref&lt;0.1054348712.1589641217.164424&gt;,1008030,ok,
    [{gen_server_SUITE,spec_init}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-04-29 21:57:02.543 ***</b>
=CRASH REPORT==== 29-Apr-2024::21:57:02.543085 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_global_default_timeout/2
    pid: &lt;0.248581.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.1054348712.1589641217.164424&gt;,1008030,ok,
                     [{gen_server_SUITE,spec_init}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2127)
    ancestors: [&lt;0.248562.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 4754
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during log_mf_h_SUITE:test/1 2024-04-29 22:01:38.984 ***</b>
=CRASH REPORT==== 29-Apr-2024::22:01:38.984286 ===
  crasher:
    initial call: gen_event:init_it/6
    pid: &lt;0.254325.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.1054348712.1591214081.21182&gt;,8135,
                     {ok,3},
                     [{log_mf_h_SUITE,test}],
                     []}
      in function  gen_event:terminate_server/4 (gen_event.erl, line 1522)
    ancestors: [&lt;0.254323.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 6772
    stack_size: 29
    reductions: 28459
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-04-29 22:01:47.985 ***</b>
=ERROR REPORT==== 29-Apr-2024::22:01:47.984985 ===
** Generic server <0.255592.0> terminating 
** Last message in was {nodedown,'peer_down_crash-527266-11793@63819cdec816'}
** When Server state == {peer_state,
                            #{args => ["-connect_all","false"],
                              name => "peer_down_crash-527266-11793",
                              peer_down => crash},
                            'peer_down_crash-527266-11793@63819cdec816',
                            [47,98,117,105,108,100,114,111,111,116,47,111,116,
                             112,47,69,114,108,97,110,103,32,8709,8868,8478,
                             47,101,114,116,115,45,49,52,46,50,46,52,47,98,
                             105,110,47,101,114,108],
                            ["-sname","peer_down_crash-527266-11793",
                             "-connect_all","false","-detached",
                             "-peer_detached","-user","peer","-origin",
                             "g1h3GHRlc3Rfc2VydmVyQDYzODE5Y2RlYzgxNgAD5mgAAAAAZi/5Fg=="],
                            undefined,undefined,<<>>,
                            {down,
                                {nodedown,
                                    'peer_down_crash-527266-11793@63819cdec816'}},
                            {<0.255589.0>,#Ref<0.1054348712.1591214081.34831>},
                            0,#{}}
** Reason for termination ==
** {nodedown,'peer_down_crash-527266-11793@63819cdec816'}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during dist in peer_SUITE 2024-04-29 22:01:47.985 ***</b>
=CRASH REPORT==== 29-Apr-2024::22:01:47.985508 ===
  crasher:
    initial call: peer:init/1
    pid: <0.255592.0>
    registered_name: []
    exception exit: {nodedown,'peer_down_crash-527266-11793@63819cdec816'}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2304)
    ancestors: [<0.255589.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 11202
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-04-29 22:01:49.417 ***</b>
=WARNING REPORT==== 29-Apr-2024::22:01:48.415800 ===
Process <0.91.0> on node 'peer_SUITE-shutdown_stop-450147-11793@63819cdec816' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-04-29 22:01:49.698 ***</b>
=WARNING REPORT==== 29-Apr-2024::22:01:49.698013 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-351045-11793@63819cdec816'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-04-29 22:01:49.782 ***</b>
=WARNING REPORT==== 29-Apr-2024::22:01:49.781964 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-351013-11793@63819cdec816'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-04-29 22:02:07.343 ***</b>
=WARNING REPORT==== 29-Apr-2024::22:02:06.341957 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-527906-11793@63819cdec816' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-04-29 22:02:07.855 ***</b>
=WARNING REPORT==== 29-Apr-2024::22:02:07.855016 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-351301-11793@63819cdec816'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-04-29 22:02:07.935 ***</b>
=WARNING REPORT==== 29-Apr-2024::22:02:07.935007 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-425956-11793@63819cdec816'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-04-29 22:02:10.581 ***</b>
=ERROR REPORT==== 29-Apr-2024::22:02:10.581093 ===
** Generic server <0.255960.0> terminating 
** Last message in was {'EXIT',<0.255955.0>,
                               {#Ref<0.1054348712.1583874050.182002>,1534065,
                                ok,
                                [{peer_SUITE,cntrl_channel_handler_crash}],
                                []}}
** When Server state == {peer_state,#{name =>
                                          "peer_SUITE-cntrl_channel_handler_crash-528674-11793",
                                      connection => standard_io},
                                    'peer_SUITE-cntrl_channel_handler_crash-528674-11793@63819cdec816',
                                    [47,98,117,105,108,100,114,111,111,116,47,
                                     111,116,112,47,69,114,108,97,110,103,32,
                                     8709,8868,8478,47,101,114,116,115,45,49,
                                     52,46,50,46,52,47,98,105,110,47,101,114,
                                     108],
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash-528674-11793",
                                     "-user","peer"],
                                    #Port<0.3120>,undefined,<<>>,running,
                                    {<0.255955.0>,
                                     #Ref<0.1054348712.1591214081.38154>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.1054348712.1583874050.182002>,1534065,ok,
    [{peer_SUITE,cntrl_channel_handler_crash}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-04-29 22:02:10.581 ***</b>
=CRASH REPORT==== 29-Apr-2024::22:02:10.581714 ===
  crasher:
    initial call: peer:init/1
    pid: <0.255960.0>
    registered_name: []
    exception exit: {#Ref<0.1054348712.1583874050.182002>,1534065,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2127)
    ancestors: [<0.255955.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3120>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 6772
    stack_size: 29
    reductions: 14388
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-04-29 22:02:11.611 ***</b>
=WARNING REPORT==== 29-Apr-2024::22:02:10.609645 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-528898-11793@63819cdec816' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-04-29 22:02:11.776 ***</b>
=WARNING REPORT==== 29-Apr-2024::22:02:11.776121 ===
peer:stop() timed out waiting for disconnect from node 'init_debug-528610-11793@63819cdec816'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-04-29 22:02:12.085 ***</b>
=WARNING REPORT==== 29-Apr-2024::22:02:12.084979 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-351461-11793@63819cdec816'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-04-29 22:02:12.106 ***</b>
=WARNING REPORT==== 29-Apr-2024::22:02:12.106856 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-450819-11793@63819cdec816'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-04-29 22:02:13.113 ***</b>
=ERROR REPORT==== 29-Apr-2024::22:02:13.113014 ===
** Generic server <0.256000.0> terminating 
** Last message in was {'EXIT',<0.255963.0>,
                           {#Ref<0.1054348712.1591214081.38246>,4061146,ok,
                            [{peer_SUITE,
                                 cntrl_channel_handler_crash_old_release}],
                            []}}
** When Server state == {peer_state,#{args =>
                                          ["-setcookie",
                                           "JJWGRJGDTWBJQFYVHUSJ","-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                           "-env","ERL_CRASH_DUMP",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@63819cdec816.2024-04-29_21.46.29/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-528802-11793",
                                           "-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@63819cdec816.2024-04-29_21.46.29/make_test_dir.stdlib_test.logs/run.2024-04-29_21.47.05/log_private/"],
                                      env => [{"ERL_AFLAGS",false}],
                                      name =>
                                          "peer_SUITE-cntrl_channel_handler_crash_old_release-528802-11793",
                                      connection => standard_io,
                                      exec =>
                                          "/usr/local/lib/erlang-25.3.2.11/bin/erl",
                                      start_cover => false},
                                    'peer_SUITE-cntrl_channel_handler_crash_old_release-528802-11793@63819cdec816',
                                    "/usr/local/lib/erlang-25.3.2.11/bin/erl",
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash_old_release-528802-11793",
                                     "-setcookie","JJWGRJGDTWBJQFYVHUSJ",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                     "-env","ERL_CRASH_DUMP",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@63819cdec816.2024-04-29_21.46.29/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-528802-11793",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@63819cdec816.2024-04-29_21.46.29/make_test_dir.stdlib_test.logs/run.2024-04-29_21.47.05/log_private/",
                                     "-user","peer"],
                                    #Port<0.3148>,undefined,<<>>,running,
                                    {<0.255963.0>,
                                     #Ref<0.1054348712.1591214081.41142>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.1054348712.1591214081.38246>,4061146,ok,
    [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-04-29 22:02:13.113 ***</b>
=CRASH REPORT==== 29-Apr-2024::22:02:13.113739 ===
  crasher:
    initial call: peer:init/1
    pid: <0.256000.0>
    registered_name: []
    exception exit: {#Ref<0.1054348712.1591214081.38246>,4061146,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2127)
    ancestors: [<0.255963.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3148>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 17731
    stack_size: 29
    reductions: 18966
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-04-29 22:12:19.772 ***</b>
=ERROR REPORT==== 29-Apr-2024::22:12:19.772351 ===
** Generic server <0.302302.0> terminating 
** Last message in was {'EXIT',<0.302299.0>,
                               {#Ref<0.1054348712.1621360641.65990>,351271,
                                [{child1,<0.302308.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.302302.0>,supervisor_SUITE},
                            rest_for_one,
                            {[child1],
                             #{child1 =>
                                   {child,<0.302308.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576459202],
                            0,never,supervisor_SUITE,
                            {ok,{{rest_for_one,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.1054348712.1621360641.65990>,351271,
    [{child1,<0.302308.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-04-29 22:12:19.772 ***</b>
=ERROR REPORT==== 29-Apr-2024::22:12:19.772352 ===
** Generic server <0.302301.0> terminating 
** Last message in was {'EXIT',<0.302299.0>,
                               {#Ref<0.1054348712.1621360641.65990>,351271,
                                [{child1,<0.302308.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.302301.0>,supervisor_SUITE},
                            one_for_all,
                            {[child1],
                             #{child1 =>
                                   {child,<0.302307.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576459202],
                            0,never,supervisor_SUITE,
                            {ok,{{one_for_all,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.1054348712.1621360641.65990>,351271,
    [{child1,<0.302308.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-04-29 22:12:19.772 ***</b>
=CRASH REPORT==== 29-Apr-2024::22:12:19.772815 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.302302.0>
    registered_name: []
    exception exit: {#Ref<0.1054348712.1621360641.65990>,351271,
                     [{child1,<0.302308.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2127)
    ancestors: [<0.302299.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 11937
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-04-29 22:12:19.773 ***</b>
=CRASH REPORT==== 29-Apr-2024::22:12:19.773080 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.302301.0>
    registered_name: []
    exception exit: {#Ref<0.1054348712.1621360641.65990>,351271,
                     [{child1,<0.302308.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2127)
    ancestors: [<0.302299.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 11905
  neighbours:


</pre></div>
<pre>
</pre>
<center>
<br /><hr /><p>
<a href="../../../all_runs.html">Test run history
</a>  |  <a href="../../../index.html">Top level test index
</a>  |  <a href="../../../suite.log.latest.html">Latest test result</a>
</p>
<div class="copyright">Copyright &copy; 2024 <a href="http://www.erlang.org">Open Telecom Platform</a><br />
Updated: <!--date-->Mon Apr 29 2024 21:47:05<!--/date--><br />
</div>
</center>
</body>
</html>
